broker/lib/jobs/BnRStatusPollerJob.js

```diff
@@ -169,5 +291,19 @@ class BnRStatusPollerJob extends BaseJob {
         throw err;
       });
   }
+
+  static updateEventMesh(instanceInfo, operation, operationStatusResponse) {
```

Comment:
Can we put the this in a utility or directly call eventmesh.server.updateAnnotationState() instead of wrapping in a separate method. As we are only adding one logger statement inside the method apart from eventmesh.server.updateAnnotationState